### PR TITLE
[MOSIP-35490] moved required helm and deploy scripts to modular repo

### PR DESCRIPTION
**Pull request by ckm007**

### CODE REVIEW

**Review by ashok-ksharma — APPROVED** [review]

File: helm/dslrig/values.yaml [hunk 1]
```diff
@@ -0,0 +1,583 @@
...
+
+  secrets:
+
+enable_insecure: false
+
+containers:
+  dslorchestrator:
+    image:
+      registry: docker.io
+      repository: syedsalman041997/dsl-orchestrator
```
Comment (Prafulrakhade) [inline helm/dslrig/values.yaml:417]: repository: mosip/dsl-orchestrator

File: helm/dslrig/values.yaml [hunk 2]
```diff
@@ -0,0 +1,583 @@
...
+         done;
+         echo "Sleeping for 30 seconds";
+         sleep 30;
+         ./entrypoint.sh
+         bash scripts/del.sh  ## Todo: Requires get api to shutdown authdemo and packetcreator. Currently we are stopping via kubectl command.
+         '
+  authdemo:
+    image:
+      registry: docker.io
+      repository: mosipdev/authentication-demo-service
```
Comment (Prafulrakhade) [inline helm/dslrig/values.yaml:473]: repository: mosipqa/authentication-demo-service

File: helm/dslrig/values.yaml [hunk 3]
```diff
@@ -0,0 +1,583 @@
...
+          port: 8082
+        initialDelaySeconds: 30
+        periodSeconds: 10
+        timeoutSeconds: 5
+        failureThreshold: 6
+        successThreshold: 1
+  packetcreator-data:
+    image:
+      registry: docker.io
+      repository: syedsalman041997/dsl-packetcreator
```
Comment (Prafulrakhade) [inline helm/dslrig/values.yaml:529]: repository: mosipqa/dsl-packetcreator

File: helm/dslrig/values.yaml [hunk 4]
```diff
@@ -0,0 +1,583 @@
...
+        ; if [ $? -gt 0 ]; then echo "Failed to add SSL certificate for host $host;
+        EXITING"; exit 1; fi; cp /usr/local/openjdk-11/lib/security/cacerts /cacerts;
+        fi
+    env:
+      - name: ENABLE_INSECURE
+        value: "true"
+    envFrom:
+      - configMapRef:
+          name: global
+    image: docker.io/openjdk:11-jre
```
Comment (Prafulrakhade) [inline helm/dslrig/values.yaml:239]: is this correct ?

**Review by Prafulrakhade — APPROVED** [review]

**Pull request merged**